app/web/features/communities/events/DiscoverEventsList.tsx

```diff
@@ -65,7 +65,7 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     justifyContent: "space-between",
     width: "100%",
-    [theme.breakpoints.down("sm")]: {
+    [theme.breakpoints.down("md")]: {
```

Comment:
FYI this was also a change that their migration codemod automatically did.  I think before this was "including small and below" and now it's "below medium".